### PR TITLE
Added more info to the email body

### PR DESCRIPTION
**Pull request by prabhukiran9999**

### CODE REVIEW

File: billing-report-utility/templates/email_body.jinja2 [hunk 1]
```diff
@@ -1,9 +1,11 @@
 Hello {{billing_group_email}},
 
-Please find attached your billing reports for the period from {{ start_date.strftime('%d-%m-%Y') }} to {{ end_date.strftime('%d-%m-%Y') }}.
+Please find attached your weekly billing reports for the period from {{ start_date.strftime('%d-%m-%Y') }} to {{ end_date.strftime('%d-%m-%Y') }}.
```
Comment (wrnu) [inline billing-report-utility/templates/email_body.jinja2:3]: Is weekly the only time period that this utility sends suing this template? I think the utility can be configured for different time periods. If that is the case then we should make this dynamic in the template to mach the configuration.

cc: @MaxWardle

Comment (MaxWardle) [reply billing-report-utility/templates/email_body.jinja2:3]: Yeah that's a good point Warren, we may have to make separate email templates or make this a variable. Just a note that I think usually the only billing reports that we send unrefined like this are the weekly ones. I think the other ones (quarterly, fiscal) go through some refinement before being sent out.

**Review by MaxWardle — APPROVED** [review]

File: billing-report-utility/templates/email_body.jinja2 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Hello {{billing_group_email}},
 
-Please find attached your weekly billing reports for the period from {{ start_date.strftime('%d-%m-%Y') }} to {{ end_date.strftime('%d-%m-%Y') }}.
+Please find attached your billing report for the period from starting at $0 CAD on {{ start_date.strftime('%d-%m-%Y') }} to ${{billing_group_total}} CAD on {{ end_date.strftime('%d-%m-%Y') }}.
```
Comment (wrnu) [inline billing-report-utility/templates/email_body.jinja2:3]: I'm not sure I understand "starting at $0 CAD on". Is the intention to convey that this is an incremental bill (non accumulative) only covering the date range?

If that is the case I think we could improve the phrasing.

Comment (prabhukiran9999) [reply billing-report-utility/templates/email_body.jinja2:3]: Yes @wrnu My intention here is to clearly specify that this is a weekly billing report. Just for the week and not the cumulative cost

Comment (wrnu) [reply billing-report-utility/templates/email_body.jinja2:3]: Maybe something like this:

> Please find attached your billing report for the period beginning on {{ start_date.strftime('%d-%m-%Y') }} and ending on {{ end_date.strftime('%d-%m-%Y') }}. Total charges for this period are ${{billing_group_total}} CAD.

I'm not sure how those dates are rendered but we should use a format specified in the [BC Gov Date and Time Standard](https://www2.gov.bc.ca/assets/gov/government/services-for-government-and-broader-public-sector/information-technology-services/standards-files/date_and_time_standard.pdf) - `YYYY-MM-DD`

Comment (prabhukiran9999) [reply billing-report-utility/templates/email_body.jinja2:3]: It Renders as DD-MM-YYYY as of now

**Review by wrnu — APPROVED** [review]

**Pull request merged**